doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -0,0 +1,36 @@
+YOLO DNNs  {#information_yolo}
```

Comment:
Please add entry for this tutorial in this file: https://github.com/opencv/opencv/blob/3.3.1/doc/tutorials/dnn/table_of_content_dnn.markdown

It should appear on [this page](http://pullrequest.opencv.org/buildbot/export/pr/10207/docs/d2/d58/tutorial_table_of_content_dnn.html) after the documentation build passes.

Reply:
doing

Reply:
Is it correct now? Sorry, I'm a beginner here ...

Reply:
> information_yolo

replace to 'dnn_yolo' (it is a label name)